test/common/testlib.py

```diff
@@ -202,7 +202,7 @@ def __enter__(self):
             def __exit__(self, type, value, traceback):
                 browser.phantom.timeout = self.timeout
         r = WaitParamsRestorer(self.phantom.timeout)
-        self.phantom.timeout = max(timeout, self.phantom.timeout)
+        self.phantom.timeout = timeout
```

Comment:
This is correct, but i'm concerned it will lead to more flakes. Maybe doesn't matter.

Reply:
This could happen of course; but then it seems much less confusing to bump the timeouts (or just drop the `with_timeout()` wrapper at all).

Reply:
FTR, I did have to retry some flaky tests, but none were due to these reduced timeouts.